### ISSUE TITLE
direct messages option for third-column

### ISSUE THREAD

**Issue by hoodiek** [tootsuite/mastodon#709]
this, plus an ability to hide direct messages in the home timeline and in notifs column even, allows users to switch to functionality they might view more intuitive

if this is implemented, i would also suggest a (number of dms since last opened if does not display in notifications or home columns) when at the getting-started page

**Comment by rixx** [comment]
A separate column für DMs would be really good – at the moment I have no good way to find old conversations with people except by scrolling down in their profiles. Depending on the person, that might be somewhere in between inconvenient and neigh impossible.

If I see this correctly, this kind of depends on #3255?

**Comment by Gargron** [comment]
I'd say this is a duplicate of #3255
